lib/features/create/create_finish/create_finish_screen.dart

```diff
@@ -128,7 +128,7 @@ class _CreateFinishBottomBarModalState
                                   )
                                 ],
                               )
-                            : Text(i18n.createFinishScreen_generateOtr),
+                            : Text(i18n.createFinishScreen_generateOtr, style: TextStyle(color: Colors.white)),
```

Comment:
just curious: why did this need to happen?

Reply:
Flutter fault moving to material 3